File: 프로그래머스/K번째수.py
def solution(array, commands):
    answer = []
    for i in range(len(commands)):
        i,j,k = commands[i]
        slice_arr = array[i-1:j]
        slice_arr.sort()
        # print(slice_arr)
        return_number = slice_arr[k-1]
        # print(return_number)
        answer.append(return_number)

    return answer

File: JS_ALGO/K번째수.js
function solution(array, commands) {
  let len = commands.length;
  let answer = [];
  for (let i = 0; i < len; i++) {
    let arr = commands[i];
    let start = arr[0] - 1;
    let end = arr[1];
    let idx = arr[2] - 1;

    let slicing = array.slice(start, end).sort((a, b) => a - b);
    answer.push(slicing[idx]);
  }

  return answer;
}
